### PR TITLE
update helm config and documentation

### PR DESCRIPTION
**Pull request by jeremi**
I did run the install and noticed the documentation was out of date so I updated it. It runs well for me.

### CODE REVIEW

**Comment by jeremi** [comment]
Over the weekend the Odoo helm has been changed, so I had to update the helm templates.
Those are the changes in the helm that break this code:
https://github.com/bitnami/charts/commit/0bcac356090f8f0eb5d2ad53a7ccfeeaa633a485#diff-cd2915619fd284ca81ebd3bf58856844260eb608bb270d7f07a58621bbdc4cbbL228

**Comment by truthfool** [comment]
Thanks for the update on new changes.
@avikganguly01 can you review the PR?

**Review by avikganguly01 — APPROVED** [review]

File: helm/templates/_helpers.tpl [hunk 1]
```diff
@@ -88,7 +88,7 @@ Return the Postgresql user
 */}}
 {{- define "odoo.databaseUser" -}}
 {{- if .Values.postgresql.enabled }}
-    {{- printf "%s" .Values.postgresql.postgresqlUsername -}}
+    {{- printf "%s" .Values.postgresql.auth.username -}}
```
Comment (SubhamPramanik) [inline helm/templates/_helpers.tpl:91]: Hey @jeremi, I see you've changed the variable names here. Was there any particular reason for the change? 

And, normally we expect the PR to contain changes in the values.yml file for the corresponding variable name updates.

Comment (jeremi) [reply helm/templates/_helpers.tpl:91]: yes, they were changed in the bitnami helm, so this is why it broke this helm:
https://github.com/bitnami/charts/commit/0bcac356090f8f0eb5d2ad53a7ccfeeaa633a485#diff-cd2915619fd284ca81ebd3bf58856844260eb608bb270d7f07a58621bbdc4cbbL228

**Pull request merged**